test/AppNotificationBuilderTests/APITests.cpp

```diff
@@ -54,16 +54,19 @@ namespace Test::AppNotification::Builder
 
         TEST_METHOD(AppNotificationBuilderSetTimeStamp)
         {
-            std::wstring dateTimeFormat{ L"%d/%m/%Y %H:%M:%S" };
-            std::wistringstream ss(L"22/12/2016 01:12:10");
+            auto time{ winrt::Windows::Foundation::DateTime::clock::now() };
+            auto builder{ AppNotificationBuilder().SetTimeStamp(time) };
 
             struct tm dateTime {};
-            ss >> std::get_time(&dateTime, dateTimeFormat.c_str());
-            std::time_t time{ mktime(&dateTime) };
-            auto timeStamp{ winrt::clock::from_time_t(time) };
+            auto timeAsTimeT{ winrt::clock::to_time_t(time) };
+            localtime_s(&dateTime, &timeAsTimeT);
 
-            auto builder{ AppNotificationBuilder().SetTimeStamp(timeStamp) };
-            auto expected{ L"<toast displayTimestamp='2016-12-22T09:12:10Z'><visual><binding template='ToastGeneric'></binding></visual></toast>" };
+            std::wstringstream buffer;
+            buffer << std::put_time(&dateTime, L"%FT%T%z");
+
+            std::wstring expectedTimestamp{ buffer.str() };
+            expectedTimestamp.insert(expectedTimestamp.size() - 2 /* offset center index */, L":");
```

Comment:
did you mean to use this c_offsetIndexValue instead of literal 2? Or is it intentional becuse the unit test tries to prevent the value from accidentally changing?

Reply:
I found out I can include the header in the unit test and use the constant. Will fix.

Reply:
Actually it might not be a bad idea to use the literal in the test, depending on what you are testing for. It may be a good thing that the test breaks whenever someone changes that value....

Reply:
Going to go ahead and use the constant since that was the original intention.